_overviews/contributors/index.md

```diff
@@ -374,19 +327,7 @@ scalaVersion := crossScalaVersions.value.head
 ~~~
 
 The second line makes sbt use by default the first Scala version of the `crossScalaVersions`.
-
-Modify the CI jobs to use all the Scala versions of your build definition by using the `+` prefix, 
-when appropriate:
-
-~~~ yaml
-jobs:
-  include:
-    - stage: test
-      script: sbt +test
-    - stage: deploy
-      if: tag =~ ^v
-      script: sbt +publishSigned sonatypeRelease
-~~~
+The CI job will use all the Scala versions of your build definition.
```

Comment:
I think it would be good to also show a workflow file that cross-runs the tests (with `+test` instead of `test`).

Or, maybe we could use `+test` from the beginning (because if there is just one Scala version used in the project, it also just works)?

Reply:
I changed the ci job to run `sbt +test` instead of `sbt test`